### ISSUE TITLE
Capture additional touch information in iOS.

### ISSUE THREAD

**Issue by bakercp** [openframeworks/openFrameworks#5643]
Currently `ofxiOSEAGLView` doesn't capture all available touch information (and doesn't fill out our `ofTouchEventArgs`).  Here's an example of a fix.  I'm happy to issue a pull request, but I wanted to see if anyone else is working on it already.

```c++		
ofTouchEventArgs touchArgs;
		touchArgs.numTouches = [[event touchesForView:self] count];
		touchArgs.x = touchPoint.x;
		touchArgs.y = touchPoint.y;
		touchArgs.id = touchIndex;

		float touchDiameter = [touch majorRadius] * 2;

		touchArgs.width = touchDiameter;
		touchArgs.height = touchDiameter;
		touchArgs.majoraxis = touchDiameter;
		touchArgs.minoraxis = touchDiameter;
		touchArgs.pressure = [touch force] / [touch maximumPossibleForce];
		touchArgs.type = ofTouchEventArgs::move;
		ofNotifyEvent(window->events().touchMoved, touchArgs);
```

A related effort by @borg is here and it seems we should also incorporate that into the next release.

https://github.com/openframeworks/openFrameworks/pull/4801

**Comment by danoli3** [comment]
Any headway on this @bakercp ? ofxPointer stuff looked like a great idea

**Comment by bakercp** [comment]
Definitely. I'm pushing ofxpointer stuff today and just added support for Apple pencil, pressure and angles and coalesced touches on ios. I would love to get it into the core, but haven't submitted much to the core lately because core PRs tend to gather dust without significant backing. If your are interested in helping move it through I'm all for it! Until then, I'm trying to continue improving ofxPointer as a drop in replacement for the core mouse/touch system.

**Comment by danoli3** [comment]
Awesome! Yeah get it up and I'll test it and lets get it in

**Comment by danoli3** [comment]
Bump :D!

**Comment by CharlesHolbrow** [comment]
I'd like to see this in core too! Is that still on the table?

**Comment by bakercp** [comment]
In the last week, I've significantly updated ofxPointer to fully support all of this stuff (including Apple Pencil features, etc).

It now supports a host of `UITouch` feature on iOS including coalesced and predicted touches, pressure, tilt, estimated property updates, etc.

https://github.com/bakercp/ofxPointer

**Comment by bakercp** [comment]
@arturoc any thoughts on this?

**Comment by ofTheo** [comment]
I wonder if this could be part of an ofxiOS update? 

There are some quite old parts in ofxiOS at the moment and @jvcleave was discussing doing a good cleanup. I would be happy to help with an ofxiOS push to modernize some of the api.

**Comment by arturoc** [comment]
no idea about iOS, @danoli3 has been usually mantaining that and taking all the decisions on what to do on that part of the code whatever he or anyone else who takes over iOS from now on is ok for me
